deploy/olm-catalog/kubevirt-hyperconverged/1.1.0/kubevirt-hyperconverged-operator.v1.1.0.clusterserviceversion.yaml

```diff
@@ -1445,7 +1445,7 @@ spec:
                 - name: CONVERSION_CONTAINER
                   value: quay.io/kubevirt/kubevirt-v2v-conversion:v2.0.0
                 - name: VMWARE_CONTAINER
-                  value: quay.io/kubevirt/kubevirt-vmware:v2.0.0-3
+                  value: quay.io/kubevirt/kubevirt-vmware:v2.0.0-4
```

Comment:
latest versions of the CSV on master is 1.2.0 (1.1.0 is the current one on release-2.4 branch). Did you run hack/build-manifests.sh after updating hack/config ?

Reply:
no, will do